src/JupyterLibrary/clients/jupyterlab/Shell.resource

```diff
@@ -61,9 +61,17 @@ Open With JupyterLab Menu
     END
 
 Maybe Accept A JupyterLab Prompt
-    [Documentation]    Click the accept button in a JupyterLab dialog (if one is open).
-    ${accept} =    Get WebElements    css:${JLAB CSS ACCEPT}
-    Run Keyword If    ${accept}    Click Element    ${accept[0]}
+    [Documentation]    Click the last button in a JupyterLab dialog (if one is open).
+    # Closing all tabs at once might create a pop-up for each tab. Let's get the
+    # Number of open tabs and try closing popups for each one.
+    ${jl_tabs} =    Get WebElements    xpath:${JL_TABBAR_NOT_SELECTED_XPATH}
+    ${len} =    Get Length    ${jl_tabs}
+    FOR    ${index}    IN RANGE    0    2+${len}
+      # Check if a popup exists
+      ${accept} =    Get WebElements    xpath:${JL_POPUP}
+      # Click the right most button of the popup
+      Run Keyword If    ${accept}    Click Element    xpath:${JL_POPUP}/button[last()]
```

Comment:
the ordering of buttons is not as indicative as their semantic classes. the "blue" one should have the `jp-mod-accept` or whatever.

Reply:
This is not enough to correctly identify the button we want to use. As an example, the popup for saving changes to a file has the `jp-mod-accept` class on both the `Discard` and `Save` buttons.
![accept](https://user-images.githubusercontent.com/82518733/147923955-73841f4f-3da0-4317-8373-5dfe9703b994.png)

Reply:
I have kept using the position of the button for now, since that is consistent in all cases I've seen. Perhaps we should raise a bug to jupyter for the usage of `jp-mod-accept`?